db/migrate/20230505213955_create_shoutout_recipients.rb

```diff
@@ -0,0 +1,10 @@
+class CreateShoutoutRecipients < ActiveRecord::Migration[7.0]
+  def change
+    create_table :shoutout_recipients do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :shoutout, null: false, foreign_key: true
+
+      t.timestamps
+    end
```

Comment:
@VitaliiShevchenko if you want to change name the table you need to rollback your previous migration and than make changes. Don't write new migration because you create 2 new tables with different names `shoutout_recipients` and `recipient_shoutouts`, it's confused.

Reply:
I did so such before, make rollback , delete and create new table shoutout_recipients but  when I try to experimented with RubyMine refactor option and after that back probably I returned recipient_shoutouts table back. Today I saw it although she is should not was. I'm appologize